image-patching/stopgap-script/nuage_overcloud_full_patch.py

```diff
@@ -311,7 +275,9 @@ def image_patching(args):
 
     workingDir = os.path.dirname(os.path.abspath(__file__))
 
-    if not args.no_signing_key and args.RpmPublicKey == None:
+    copy_repo_file(args.ImageName, args.RepoFile)
```

Comment:
can we stick to previous approach, where we create repo file after uninstalling OVS?